include/votca/tools/akimaspline.h

```diff
@@ -98,8 +98,8 @@ inline double AkimaSpline::getSlope(double m1, double m2, double m3,
   if ((m1 == m2) && (m3 == m4)) {
```

Comment:
We should fix this comparison, too.

Reply:
Wow you are fast. I just created this.

Reply:
And what would be a suitable tolerance? I have no idea what sizes these numbers typically have.